### ISSUE TITLE
adding hearts to custom recipes

### ISSUE THREAD

**Issue by ghost** [dewPrzemuS/P-LifeSteal#7]
is it possible to add a heart item to the revive book recipe? and if it currently isnt possible can it be a new feature?

**Comment by nicolbolas123** [comment]
yes but what do i change it to? i try things like extra_heart which its called but it doesnt work for me
